tutorials/model-guide.md

```diff
@@ -0,0 +1,120 @@
+# AI Catalog Model Tech Doc Guide
+
+This guide can be used as a reference point when trying to add/customize your own documentation for individual models.
+
+## Template
+
+- [index.md](#table-of-contents---indexmd)
+- [model.md](#model-information---modelmd)
+- [code/usage-examples.md](#model-usage---codeusage-examplesmd)
+- [training.md](#training-information---trainingmd)
+- [ethics.md](#ethical-considerations---ethicsmd)
+- [license.md](#license---licensemd)
+
+### Table of Contents - index.md
+
+Red Hat Developer Hub provides a Table of Contents on the left-hand side of the Tech Docs page, however, when you first open Tech Docs it may be helpful to display that information right in the face of the user. This ensures that the index.md file for all models is consistent in what shows to the user.
+
+### Model Information - model.md
+
+#### Model Description
+
+This section should contain a brief description/introduction of the model. Pretty much every model will have something along the lines of this provided. Additionally, you should aim to provide the following information as part of this document:
+
+1. Developed By
+2. Code Repository
+   1. GitHub/GitLab if provided
+3. Model Repository
+   1. Link to the repository page of the model (i.e. HuggingFace)
+4. Relevant Papers (if provided)
+5. Release Date (if provided)
+6. Licensing
+   1. Link to the license (i.e. link to the Apache 2 website)
+
+#### Intended Use
+
+This section should contain the intended use of the model. Most suppliers of the models will have a section like this on the repository page for the model. 
+
+### Model Usage - code/usage-examples.md
+
+This document should aim to provide users with the information required to consume the model. For example if the model has a Python package that can be imported then it would be important to provide examples of different actions you can perform.
```

Comment:
```suggestion
This document should aim to provide users with the information required to consume the model. For example, if the model has a Python package that can be imported then it would be important to provide examples of different actions you can perform.
```